query-node/joystream-query-node/bootstrap/members.ts

```diff
@@ -1,4 +1,4 @@
-import { Membership } from '../generated/indexer/entities/Membership';
+import { Membership } from '../generated/graphql-server/src/modules/membership/membership.model';
```

Comment:
shouldn't we assume that the graphql-server may be on a different machine and is decoupled from the indexer?

Reply:
Yes, that is right but we are using graphql server models now and that means we depend on. Maybe we can copy model files to indexer in `codegen` command but I am not sure if that is the right way. Do you have any suggestions here?

Reply:
Simply copying is the easiest and I think it's been working so far, but perhaps we can package it into a module and import? The main advantage would be versioning, otherwise I think copying is fine